doc/yaml.md

```diff
@@ -412,3 +412,4 @@ $ ys -e 'say: .[1, 2, 3]'
 ```
 
 Don't worry, you'll get the hang of it quickly!
+
```

Comment:
Maybe it is best to not explain the escape `.` method at all here. The change between code-mode and data-mode is already an involved concept to grasp. I think one can get vary far without the escape `.` trick. Having a "Gotcha" cookbook like this and at the same time showing how to use data-mode to resolve them is a very good vehicle to introduce data-mode. Mixing the escape trick into it is too much, I think.